server/src/test/java/io/crate/integrationtests/SysClusterSettingsTest.java

```diff
@@ -21,7 +21,7 @@
 
 package io.crate.integrationtests;
 
-import static io.crate.testing.Asserts.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Is there a possibility to prioritize our assert over the assertj one?

Reply:
I don't think so